config/site_settings.yml

```diff
@@ -239,6 +239,20 @@ basic:
       - twitter
       - facebook
       - email
+  desktop_category_page_sort_order:
+    client: true
+    type: enum
+    default: default
+    choices:
+      - default
```

Comment:
I do wonder if `default` should be named slightly differently, what is the default ordered by?

Reply:
Oh I saw it was bump date, so this option should be `bump_date` IMO